pages/am/3.x/user-guide/branding/user-guide-branding.adoc

```diff
@@ -4,10 +4,22 @@
 :page-folder: am/user-guide
 :page-layout: am
 
-AM can be customized to suit your organization's brand requirements and user expectations.
-AM branding gives you a consistent user experience when navigating between customer applications and AM server.
+== Overview
 
-You can create customized templates for individual applications or across all applications in a security domain.
+Since the dawn of centralized IAM, moderns apps have moved away from login directly in the app. This means however that it is critical that you offer a consistent user experience when redirecting users to the login scenarios in AM. The user should feel confindent and secure. An inconsitent or unclear user experience might affect customer retention and perception of your company.
```

Comment:
```suggestion
Since the dawn of centralized IAM, modern apps have moved away from logins directly in the app, so it is critically important to offer a consistent user experience when redirecting users to the login scenarios in AM. Users should feel confindent and secure and an inconsistent or unclear user experience might affect your organization's customer retention and reputation
```